tcms/kiwi_auth/tests/test_views.py

```diff
@@ -329,6 +364,30 @@ def test_get_template_names(self):
 class TestPasswordResetView(TestCase):
     """Test for password reset view"""
 
+    def setUp(self):
+        self.password_reset_url = reverse("tcms-password_reset")
+
     def test_form_class(self):
-        response = self.client.get(reverse("tcms-password_reset"))
-        self.assertIsInstance(response.context["form"], forms.PasswordResetForm)
+        response = self.client.get(self.password_reset_url)
+        self.assertEqual(
+            str(type(response.context["form"])),
+            str(forms.PasswordResetForm),
```

Comment:
`type()` returns a class and all classes have `.__name__` attribute. Does that work instead of converting to string ?

Reply:
`.__name__` attribute return `PasswordResetForm` which doesn't makes it clear from where it is imported. When I use `str`it returns `<class 'tcms.kiwi_auth.forms.PasswordResetForm'>` which ensures it is overriden